dwitter/templates/dweet/dweet.html

```diff
@@ -175,6 +174,10 @@
     {% load staticfiles %}
     {% compress js %}
     <script src="{% static "libs/stats.min.js" %}"></script>
+    <script src="{% static "libs/gif.js" %}"></script>
+    <script src="{% static "libs/loopbuster.js" %}"></script>
+    <script src="{% static "libs/loopstopper.js" %}"></script>
+    <script src="{% static "libs/jquery.min.js" %}"></script>
```

Comment:
Hmm... Is jquery really required? This loads jquery for every single dweet and also extends the dweet language to have access to jquery functions

Reply:
That one's for the resize polyfill, but I think I know of a way to remove it